### DISCUSSION TITLE
Our school does not provide students with official educational email addresses. But the reason why I was rejected was that I did not provide an official email to apply for authentication

### DISCUSSION THREAD

**Discussion by jsonza6** [community/community#53798] [GitHub Education]
I used a camera to provide my admission letter and student ID for student authentication, but why are they still frequently rejected? Is it because all my files are in Chinese😣😣

**Comment by samus-aran** [comment]
Hi @zwq3107016156 👋 

> Is it because all my files are in Chinese😣😣

That's the most likely cause 👍 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

Let us know how you get on 👍

**Reply by jsonza6** [reply]
![IMG_20230428_001342](https://user-images.githubusercontent.com/88477295/234931854-51b079b0-922d-486b-b005-ce2e407e742a.jpg)
I uploaded photos of my student ID card, student card, and admission letter with English annotations, but my application was still rejected. The reason given was 'Attendees of this school must verify and use their academic email address.' The problem now is that my school only provides official educational email addresses for teachers, not for students. Therefore, I can only apply for verification using my own personal email address.

**Comment by hitzaki** [comment]
申请成功了吗

**Reply by jsonza6** [reply]
![IMG_20230428_001342](https://user-images.githubusercontent.com/88477295/234925755-58deef21-420b-4389-98a5-827e4f6a808a.jpg)

I was not able to pass the application. This is simply ridiculous. I was rejected seven or eight times. My school clearly does not provide education email for students, but it is absurd that GitHub has been refusing me continuously, insisting that I must use the email provided by the school to verify my identity. I really don't understand. Obviously, I uploaded my student ID card, student card, even my transcript and admission letter, and provided English annotations, but I was still refused. I am truly speechless 😓

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack

**Reply by jsonza6** [reply]
这几天我尝试了使用学信网验证报告并且附上了英文注释，改了GitHub我的简介和姓名。
拒绝理由仍然是：我必须使用学校提供的官方邮箱进行验证😂😂 我已经放弃了。
